init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/loader/Loader.java

```diff
@@ -83,7 +83,7 @@ public void run(String... args) throws Exception {
             var sites = trialConfig.getSites();
 
             initProgressReporter.submitProgress(LoaderProgress.CREATE_SITES.message());
-            List<String> siteIds = siteServiceInvoker.execute(sites);
+            List<String> siteIds = siteServiceInvoker.createManySites(sites);
```

Comment:
why createManySites? .createSites would be the usual approach? Do we expect there to be a createNotManySites ?

Reply:
I agree, changed it to .createSites. In addition, to keep the convention I change createManyRoles to createRoles as well in the other invoker, so these files were added to this PR but it is only about name changing ang nothing more